### PR TITLE
Separate hadoop and native batch docs more

### PR DESCRIPTION
**Pull request by jon-wei**
Slight adjustment to batch ingestion docs, creating a separate page for hadoop and restructuring the existing batch ingestion page to be a directory page pointing to specific task type docs

### CODE REVIEW

**Comment by fjy** [comment]
👍

File: docs/content/ingestion/batch-ingestion.md [hunk 1]
```diff
@@ -6,342 +6,19 @@ layout: doc_page
...
-Druid works out of the box with many Hadoop distributions.
-
-If you are having dependency conflicts between Druid and your version of Hadoop, you can try
-searching for a solution in the [Druid user groups](https://groups.google.com/forum/#!forum/druid-
-user), or reading the Druid [Different Hadoop Versions](../operations/other-hadoop.html) documentation.
+Hadoop can be used for batch ingestion. The Hadoop-based batch ingestion will be faster and more scalable than the native batch ingestion. See [here](../ingestion/hadoop.html) for more details.
 
 ## Command Line Hadoop Indexer
 
 If you don't want to use a full indexing service to use Hadoop to get data into Druid, you can also use the standalone command line Hadoop indexer.
```
Comment (dclim) [inline docs/content/ingestion/batch-ingestion.md]: I feel like this 'option' for the command line Hadoop indexer makes more sense as a note at the beginning of the Hadoop indexing page.

File: docs/content/toc.md [hunk 2]
```diff
@@ -17,6 +17,8 @@ layout: toc
   * [Schema Design](/docs/VERSION/ingestion/schema-design.html)
   * [Schema Changes](/docs/VERSION/ingestion/schema-changes.html)
   * [Batch File Ingestion](/docs/VERSION/ingestion/batch-ingestion.html)
+    * [Native Batch Ingestion](docs/VERSION/native-batch.html)
```
Comment (dclim) [inline docs/content/toc.md]: /ingestion/native-batch.html?

File: docs/content/toc.md [hunk 3]
```diff
@@ -17,6 +17,8 @@ layout: toc
...
+    * [Hadoop Batch Ingestion](docs/VERSION/hadoop.html)
```
Comment (dclim) [inline docs/content/toc.md]: /ingestion/hadoop.html?

File: docs/content/ingestion/native-batch.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+For Roaring bitmaps:
+
+|Field|Type|Description|Required|
+|-----|----|-----------|--------|
+|type|String|Must be `roaring`.|yes|
+|compressRunOnSerialization|Boolean|Use a run-length encoding where it is estimated as more space efficient.|no (default == `true`)|
+
+## Segment publishing modes
+
+While ingesting data using the Index task, it creates segments from the input data and publishes them. For segment publishing, the Index task supports two segment publishing modes, i.e., _bulk publishing mode_ and _incremental publishing mode_ for [perfect rollup and best-effort rollup](./design/index.html), respectively.
```
Comment (dclim) [inline docs/content/ingestion/native-batch.md]: Should be `../design` instead of `./design`

File: docs/content/ingestion/batch-ingestion.md (hunk 1 above)
Comment (jon-wei) [reply docs/content/ingestion/batch-ingestion.md]: Moved this to the beginning of the hadoop page

File: docs/content/toc.md (hunk 2 above)
Comment (jon-wei) [reply docs/content/toc.md]: thanks, fixed

File: docs/content/toc.md (hunk 3 above)
Comment (jon-wei) [reply docs/content/toc.md]: fixed

File: docs/content/ingestion/native-batch.md (hunk 4 above)
Comment (jon-wei) [reply docs/content/ingestion/native-batch.md]: i updated this in `ingestion/native_tasks.md` (current page for native batch indexing with the parallel index task)

**Pull request merged**